Source/SwiftLintFramework/Rules/RuleConfigurations/SuperCallConfiguration.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  SuperCallConfiguration.swift
+//  SwiftLint
+//
+//  Created by Angel Garcia on 05/09/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct SuperCallConfiguration: RuleConfiguration, Equatable {
+    var defaultIncluded = [
+        //NSObject
+        "awakeFromNib()",
+        "prepareForInterfaceBuilder()",
+        //UIViewController
+        "transitionCoordinator()",
+        "viewDidLoad()",
+        "viewWillAppear(_:)",
+        "viewWillDisappear(_:)",
+        "viewDidAppear(_:)",
+        "viewDidDisappear(_:)",
+        "addChildViewController(_:)",
+        "removeFromParentViewController()",
+        "transition(from:to:duration:options:animations:completion:)",
+        "transitionFromViewController(_:toViewController:duration:options:animations:completion:)",
+        "didReceiveMemoryWarning()",
+        "encodeRestorableState(with:)",
+        "encodeRestorableStateWithCoder(_:)",
+        "decodeRestorableState(with:)",
+        "decodeRestorableStateWithCoder(_:)",
+        "setEditing(_:animated:)",
+        //UIView
+        "updateConstraints()",
+        "prepareForReuse()",
+        //UICollectionViewLayout
+        "invalidateLayout()",
+        "invalidateLayout(with:)",
+        "invalidateLayoutWithContext(_:)",
+        //XCTestCase
+        "tearDown()",
+        "setUp()"
```

Comment:
I haven't seen anywhere explicitly, but the default template includes calling super: https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/04-writing_tests.html#//apple_ref/doc/uid/TP40014132-CH4-SW2